contributions/templates/admin/export.html

```diff
@@ -0,0 +1,35 @@
+{% extends "admin/import_export/base.html" %}
+{% load i18n %}
+{% load admin_urls %}
+{% load import_export_tags %}
+
+{% block breadcrumbs_last %}
+{% trans "Export" %}
```

Comment:
```suggestion
{% translate "Export" %}
```
Is there a difference between `trans` and `translate`? I would prefer to stick to one way.